api/envoy/config/core/v3/grpc_service.proto

```diff
@@ -43,6 +43,12 @@ message GrpcService {
     string authority = 2
         [(validate.rules).string =
              {min_len: 0 max_bytes: 16384 well_known_regex: HTTP_HEADER_VALUE strict: false}];
+
+    // Indicates the retry policy for re-establishing the gRPC stream
+    // This field is optional. If max interval is not provided, it will be set to ten times the provided base interval.
+    // Currently only supported for xDS gRPC streams.
+    // If not set, xDS gRPC streams default base interval:500ms, maximum interval:30s will be applied.
+    RetryPolicy retry_policy = 3;
```

Comment:
nit: just realized there are 2 RetryPolicy types, so let's make this explicit: `config.core.v3.RetryPolicy`.

Reply:
done, thanks

Reply:
reverted the change, it looks like the linter does not like it

```
INFO: Analyzed target @com_github_bufbuild_buf//:bin/buf (1 packages loaded, 1 target configured).
INFO: Found 1 target...
WARNING: /build/tmp/_bazel_envoybuild/b570b5ccd0454dc9af9f65ab1833764d/external/com_github_bufbuild_buf/BUILD.bazel:6:10: @com_github_bufbuild_buf//:bin/buf is a source file, nothing will be built for it. If you want to build a target that consumes this file, try --compile_one_dependency
INFO: Elapsed time: 0.938s, Critical Path: 0.04s
INFO: 1 process: 1 internal.
INFO: Running command line: /build/tmp/_bazel_envoybuild/b570b5ccd0454dc9af9f65ab1833764d/external/com_github_bufbuild_buf/bin/buf lint
INFO: Streaming build results to: https://source.cloud.google.com/results/invocations/6a1a3f8a-844c-42b2-a403-3c2bbbc1ca17
Format check failed, try apply following patch to fix:
diff --git a/api/envoy/config/core/v3/grpc_service.proto b/api/envoy/config/core/v3/grpc_service.proto
index 7cfe602..f266c7b 100644
--- a/api/envoy/config/core/v3/grpc_service.proto
+++ b/api/envoy/config/core/v3/grpc_service.proto
@@ -48,7 +48,7 @@ message GrpcService {
     // This field is optional. If max interval is not provided, it will be set to ten times the provided base interval.
     // Currently only supported for xDS gRPC streams.
     // If not set, xDS gRPC streams default base interval:500ms, maximum interval:30s will be applied.
-    config.core.v3.RetryPolicy retry_policy = 3;
+    RetryPolicy retry_policy = 3;
   }
 
   // [#next-free-field: 9]
```
I am guessing because `RetryPolicy` and `GrpcService` are part of the same package `envoy.config.core.v3`? Please correct me here, thanks

Reply:
Sorry about the back and forth.

Reply:
No worries, thank you so much!